### ISSUE TITLE
reproduction of Micronet-M0 4GPU

### ISSUE THREAD

**Issue by HydrogenSulfate** [liyunsheng13/micronet#28]
Thanks for your great work, but I have trained the `train_micronet_m0_4gpu.sh` script, but got best Acc@1 45.86 at epoch 596, lower than 46.6 reported in your repo with pytorch 1.7.1 CUDA 10.2 cudnn 7.6.05. 

And I did not make any modifications to the training program, is there something wrong with my training program?

**Comment by liyunsheng13** [comment]
The smallest model does have some stability issues, but it should have less than 0.5% variance. I'm curious whether you can reproduce the result with the released model.

**Comment by HydrogenSulfate** [comment]
> The smallest model does have some stability issues, but it should have less than 0.5% variance. I'm curious whether you can reproduce the result with the released model.

the accuracy of the [provided model](http://www.svcl.ucsd.edu/projects/micronet/assets/micronet-m0.pth) is ok(46.69).

**Comment by HydrogenSulfate** [comment]
> The smallest model does have some stability issues, but it should have less than 0.5% variance. I'm curious whether you can reproduce the result with the released model.

one more question, why the learning rate is 10 times larger than descripted in paper? ![image](https://user-images.githubusercontent.com/23737287/186923480-d26d523c-2f38-4071-ba8f-8e5c9ec194d3.png) ![image](https://user-images.githubusercontent.com/23737287/186923718-7cdc4452-280f-4414-86a9-dfd275f94469.png)

**Comment by flytocc** [comment]
How about the stability of the largest model (m3)? @liyunsheng13

**Comment by liyunsheng13** [comment]
The largest model should be stable. In the experiments I did, the larger the model is, the stabler the performance will be.
